deploy/backplane/00-admin.namespace.yml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-backplane-admin
```

```suggestion
  name: openshift-backplane-cluster-admin
```
Clearer if we end with "cluster-admin"